packages/helix-shared-bounce/src/bounce.js

```diff
@@ -29,17 +37,41 @@ function bounce(func, { responder, timeout = 500 }) {
     context.invocation.bounceId = bounceId;
 
     // run the quick responder function
-    const holdingResponse = await responder(request, context);
+    const holdingResponse = (async () => {
+      const res = responder(request, context);
+      const wait = timer.setTimeout(timeout);
```

Comment:
I would move this in the try/catch block below, in case `responder()` already throws.

Reply:
done.